x-pack/plugins/cases/common/ui/types.ts

```diff
@@ -5,4 +5,4 @@
  * 2.0.
  */
 
-export * from '../../translations';
+export * from '../../public/containers/types';
```

Comment:
this is weird. this is a brand new file, but github thinks it comes from one of my deleted files?